docs/source/install/index.rst

```diff
@@ -157,27 +157,58 @@ containing the executables, and running a simple test command:
 
 .. _use_docker:
 
-Use a docker image
-~~~~~~~~~~~~~~~~~~
-Multiple versions of OpenFAST are available as docker images from our `docker registry <https://hub.docker.com/r/nrel/openfast>`_.
+Running OpenFAST with docker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+OpenFAST is avilable to be run on docker starting with version 3.5.3. Three approaches are shared below.
+
+Using a docker image from Docker hub
```

Comment:
Do we add that it'll be available soon? and not in this specific release?

Reply:
I thought about it, but I don't now if I'll remember we need to change that in a later release.